Test/Load/config.js

```diff
@@ -0,0 +1,15 @@
+import crypto from "k6/crypto";
+
+export const URL = "http://34.220.243.58";
```

Comment:
Can you add comments here similar to the ones added for the M1 PR?

Reply:
Sounds good, I added in the comments